File: app/src/main/java/com/artear/multitrackerandroid/trackers/type/event/MyEvent.java
package com.artear.multitrackerandroid.trackers.type.event;


import com.artear.multitracker.contract.send.TrackerEvent;

public class MyEvent implements TrackerEvent {

    private String name;

    public MyEvent(String name) {
        this.name = name;
    }

    @Override
    public String toString() {
        return "< MyEvent - name = " + name + " >";
    }
}

File: app/src/main/java/com/artear/multitrackerandroid/trackers/type/event/MyEvent.kt
package com.artear.multitrackerandroid.trackers.type.event


import com.artear.multitracker.contract.send.TrackerEvent

class MyEvent(private val name: String) : TrackerEvent {

    override fun toString(): String {
        return "< MyEvent - name = $name >"
    }
}
